flow-server/src/main/resources/plugins/stats-plugin/stats-plugin.js

```diff
@@ -103,24 +103,28 @@ function collectChunks(statsJson, acceptedChunks) {
         const modules = [];
         // Add all modules for chunk as slimmed down modules
         chunk.modules.forEach(function (module) {
-          const slimModule = {
-            id: module.id,
-            name: module.name,
-            source: module.source
-          };
-          if (module.modules) {
-            slimModule.modules = collectSubModules(module);
+          if(module.name.includes("generated-flow-imports.js") || module.name.includes("generated-flow-imports-fallback.js")) {
+            const slimModule = {
+              id: module.id,
+              name: module.name,
+              source: module.source
+            };
+            if (module.modules) {
+              slimModule.modules = collectSubModules(module);
+            }
           }
           modules.push(slimModule);
         });
```

Comment:
The push slim module needs to be inside the if else it will push undefined to modules.
```suggestion
            }
            modules.push(slimModule);
          }
        });
```